File: semestr-8/Jezyk C#/alexander/Csharp/Archiwum/2021s/Id06TC2/Command/Command/Sportsmen.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Command
{
    public class Sportsmen
    {
        private string name;
        public Sportsmen(string name) {  this.name=name; }
        public string Run() { return "Runing"; }
        public string Jump() { return "Jumping"; }
        public string Swim() { return "Swimming"; }
        public string execute(SportsmenCommand cmd)
        {
            return name + ": " + cmd.execute(this);
        }
    }
}

File: semestr-8/Programowanie obiektowe w Java/alexander/POJ/2023s/IZ08IO3/StrategyProject/src/Sportsmen.java
public class Sportsmen
{
    private final String name;
    public Sportsmen(String name) { this.name = name; }
    public String run() { return "Running"; }
    public String jump() { return "Jumping"; }
    public String swim() { return "Swimming"; }
    public String execute(SportsmenCommand cmd)
    {
        return name + ": " + cmd.execute(this);
    }
}
